src/main/kotlin/com/openlattice/indexing/BackgroundIndexingService.kt

```diff
@@ -154,6 +158,16 @@ class BackgroundIndexingService(
                 "LIMIT $FETCH_SIZE"
     }
 
+    private fun getEntityDataKeys(entitySetId: UUID): PostgresIterable<UUID> {
+        return PostgresIterable(Supplier<StatementHolder> {
+            val connection = hds.connection
+            val stmt = connection.createStatement()
+            stmt.fetchSize = 64000
```

Comment:
We already have a const val FETCH_SIZE = 128000 or is that too high? On that note, I guess getDirtyEntitiesQuery should use fetchSize and not LIMIT

Reply:
I ended changing how this is done entirely, because executing the query is super expensive even without a fetch size.